src/coffee/validator.coffee

```diff
@@ -29,18 +29,23 @@ class Validator
       delimiter: options.csvDelimiter or ','
       quote: options.csvQuote or '"'
 
-  parse: (csvString) ->
+  parse: (csvString, delimiter ) ->
```

Comment:
why you parsing `delimiter` here. 

It's not being used anywhere in this method.